drivers/spi/spi_silabs_eusart.c

```diff
@@ -15,6 +20,12 @@
 #include <zephyr/drivers/clock_control/clock_control_silabs.h>
 #include <zephyr/drivers/pinctrl.h>
 #include <zephyr/logging/log.h>
+#ifdef SPI_SILABS_SPI_EUSART_DMA
+#include <zephyr/drivers/dma.h>
+#endif
```

Comment:
Unless you have a good justification to do so, conditional includes are not a good idea, since they prevent the possibility of taking advantage of `IS_ENABLED()` in the code.